### ISSUE TITLE
Bug: calcite-tooltip not working on disabled calcite-tile

### ISSUE THREAD

**Issue by AdelheidF** [Esri/calcite-components#3173]
I want to display a tooltip for a disabled tile to tell the user why this option is not available. But the tooltip doesn't show if a tile is disabled.

### Actual Behavior

no tooltip shows

### Expected Behavior

tooltip should show

### Reproduction Steps and Sample

https://codepen.io/afreitag/pen/VwWoGVV?editors=1000

### Relevant Info

_Version_: `@esri/calcite-components@1.0.0-beta.65`

- [ ] CDN
- [ ] NPM package

**Comment by jcfranco** [comment]
**TL;DR**: `calcite-tile`'s only interactive portion is the internal link, so removing the host pointer-events could suffice.

Went on a bit of a rabbit hole. 😅

1. It looks like `disabled` is only applicable when the `href` property is set as it makes the tile interactive
2. Supporting `disabled` when non-interactive (w/o `href`) seems to contradict https://github.com/Esri/calcite-components/issues/2655#issuecomment-888588587, but there wasn't an example of 'partial-disabled' behavior during that time.
3. In the provided sample, assuming `pointer-events:none` was set on the internal container, there would still be an issue for keyboard users as the tile w/o `href` is not tabbable
4. This isn't really applicable to `calcite-tile` as it's not a form control, but the spec advises for [disabled form controls to prevent `click` events](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#enabling-and-disabling-form-controls:-the-disabled-attribute). There's no details on other events and there's even a thread about this: https://github.com/whatwg/html/issues/2368 I think we could follow this pattern for interactive components too.

@Esri/calcite-design-representatives WDYT about ☝🏼 ?

cc @benelan

**Comment by AdelheidF** [comment]
Here is a related issue to allow focus on a tile: https://github.com/Esri/calcite-components/issues/2141

**Comment by macandcheese** [comment]
@jcfranco above sounds good in combination with https://github.com/Esri/calcite-components/issues/2141

**Comment by driskull** [comment]
@AdelheidF as a workaround for now, can you add the tooltip to a div that is wrapping the disabled tile?
